hazelcast/src/test/java/com/hazelcast/splitbrainprotection/AbstractSplitBrainProtectionTest.java

```diff
@@ -216,15 +233,37 @@ protected static void initCluster(Config config, TestHazelcastInstanceFactory fa
             for (String postfix : asList("", "shutdown", "shutdownNow")) {
                 config.addDurableExecutorConfig(newDurableExecConfig(splitBrainProtectionOn, splitBrainProtectionName, postfix));
                 config.addExecutorConfig(newExecConfig(splitBrainProtectionOn, splitBrainProtectionName, postfix));
-                config.addScheduledExecutorConfig(newScheduledExecConfig(splitBrainProtectionOn, splitBrainProtectionName, postfix));
+                config.addScheduledExecutorConfig(
+                        newScheduledExecConfig(splitBrainProtectionOn, splitBrainProtectionName, postfix));
             }
             config.addSetConfig(newSetConfig(splitBrainProtectionOn, splitBrainProtectionName));
             config.addPNCounterConfig(newPNCounterConfig(splitBrainProtectionOn, splitBrainProtectionName));
         }
-
         cluster.createFiveMemberCluster(config);
+        for (SplitBrainProtectionOn splitBrainProtectionOn : types) {
```

Comment:
Is this is a DEBUG information? If so, why would you commit it to the master?

Reply:
We discussed this on slack so I will mark this as approved.